src/styles/index.scss

```diff
@@ -1,3 +1,7 @@
+:root {
+  color-scheme: light dark; // affects default colors of scrollbars
+}
+
```

Comment:
As mentioned here: https://drafts.csswg.org/css-color-adjust/#color-scheme-prop

The user agent match's the following to the user’s preferred color scheme:

1. the color of the canvas surface the initial value of the color property, and the system colors

2. the default colors of scrollbars(_which we plan to use_) and other interaction UI

3. the default colors of form controls and other "specially-rendered" elements

4. the default colors of other browser-provided UI, such as "spellcheck" underlines